numpy/distutils/fcompiler/gnu.py

```diff
@@ -193,13 +199,10 @@ def get_libraries(self):
             opt.append(g2c)
         c_compiler = self.c_compiler
         if sys.platform == 'win32' and c_compiler and \
-               c_compiler.compiler_type == 'msvc':
+                c_compiler.compiler_type == 'msvc':
             # the following code is not needed (read: breaks) when using MinGW
             # in case want to link F77 compiled code with MSVC
             opt.append('gcc')
-            runtime_lib = msvc_runtime_library()
```

Comment:
These lines actually had a signfiicant impact becuase the class is inherited. It's not just a "cleanup"